### ISSUE TITLE
Secondary AI core without power

### ISSUE THREAD

**Issue by Nostr0** [yogstation13/Yogstation#13115]
### Severity

Inconvenience

### Short Description

Secondary AI core without power nearly every round by default

## Round ID:
No certain rounds/Random

## Testmerges:

Not Supplied

## Reproduction:

1. Play AI
2. The secondary AI Data Core looses power within a few minutes of the roundstart due to the insane power consumption. Futhermore is the science testing lab possibly the worst place of all of places for the server as there are too many points where the power cables can just simply get cut off by mice.
3. Get a near death experience and a hearth attack every few minutes because of this

One antag could for example simply cut 2 wires (science and near the incinerator) and the AI will die within 10 minutes of roundstart. There should simply be an earlier warning of powerloss because when you get the message you just got a few minutes at best to restore power before you permadeath. Most people won't bother fixing cables as quickly as the AI needs the power now with the new changes.

Possible solutions:
1. Reduce power consumption a bit.
2. Place the secondary AI Data Core somewhere else

tldr: Secondary AI Data Core is offline by default most of the time

### Round ID

39139

### BYOND Username

Nostr0

**Comment by Nostr0** [comment]
Might add just on the Map "YogStation" as far as I can tell
